src/index.html

```diff
@@ -7,8 +7,88 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header-container">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            href="#apple"
+            class="nav__link is-active">
+            Apple
```

Comment:
the closing bracket should be on the same level as open bracket. the child should have 2-space offset
![image](https://user-images.githubusercontent.com/22824431/235643267-88eac6cf-c23e-4331-b319-9ca3bcfa7f42.png)
```suggestion
            class="nav__link is-active"
            >
              Apple
```

Reply:
please fix all cases